lib/std/zig/parser_test.zig

```diff
@@ -228,6 +228,23 @@ test "zig fmt: top-level enum missing 'const name ='" {
     , &[_]Error{.expected_token});
 }
 
+test "zig fmt: top-level container missing 'const name ='" {
+    // somehow testCanonical() causes these to be interpreted as
```

Comment:
I think this is kind of an awkward comment to add into the code itself. Also the line numbers in this comment will quickly be outdated with more changes to lib/std/zig/Ast.zig.

Reply:
yes i realize and plan on deleting it once i find a solution.

Reply:
Whoops, sorry for barging in! I thought this was already solved because the tests pass for me.

Reply:
no problem. thanks for testing it out.  yeah its a bit confusing whats happening. i could have been more clear.  i thought those tests should fail without the change to Ast.zig. but they dont. you can observe it by removing the change to Ast.zig and running `zig test parser_test.zig`.